desktop/apps/consignments2/templates/landing.jade

```diff
@@ -5,9 +5,11 @@ block head
 
```

Comment:
Oh, the landing page is not react (it's a static page that uses the handy `JSONPage` component), so totally separate from the submission flow.

There is a `submission.jade` which is basically an empty template with a single element that all of the react components bind to.